### PR TITLE
#166267831 Set up Babel

### PR DESCRIPTION
**Pull request by JuwonAbiola**
#### Description
Setup Babel

#### Type of change
- install @babel/core: "7.4.5"
- install @babel/node: "7.4.5"
- install @babel/preset-env: "7.4.5"
- install @babel/register: "7.4.4"

#### How Has This Been Tested?
NA

#### Checklist:
NA

#### Pivotal Tracker
[#166267831](https://www.pivotaltracker.com/n/projects/2350588/stories/166267831)

### CODE REVIEW

**Review by henryade — COMMENTED** [review]
In your PR, your pivotal story should have a link i.e `[id](link)`

**Comment by JuwonAbiola** [comment]
> In your PR, your pivotal story should have a link i.e `[id](link)`

done

**Review by Adekoreday — COMMENTED** [review]
great job, u a better approach might be to seperate the development start script server from the production server
i.e. include :
"start": "node server/index.js",
"startDev": "babel - node blah blah

**Comment by JuwonAbiola** [comment]
> great job, u a better approach might be to seperate the development start script server from the production server
> i.e. include :
> "start": "node server/index.js",
> "startDev": "babel - node blah blah

done

**Review by henryade — APPROVED** [review]
Nice one. But no .babelrc file?!

**Comment by allebd** [comment]
You will need to rebase your branch with develop

**Review by Adekoreday — COMMENTED** [review]
to resolve some merge conflicts do not override the start script 
have a script for start: node server/index.js
and another script to start the development server startDev babel-node blah blah

**Review by Lundii — APPROVED** [review]
Okay

**Comment by allebd** [comment]
There is no comma at the end of your start script before the startDev script

**Comment by allebd** [comment]
No closing bracket in your package.json file

**Review by Nennyfills — APPROVED** [review]
Good job.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env"]
+    }
```
Comment (KvNGCzA) [inline .babelrc]: Fix the indentation of this file.

**Review by KvNGCzA — CHANGES_REQUESTED** [review]

**Comment by KvNGCzA** [comment]
Also, Add nodemon and edit the start:dev script to use nodemon

File: package.json [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
-    "eslint-plugin-import": "^2.17.3"
-  }
+
+    "name": "Hello-Books-BE",
+    "version": "1.0.0",
+    "description": "A system that allows efficient, cost-effective management of our library",
+    "main": "index.js",
+    "scripts": {
+        "start": "node server/index.js",
+        "startDev": "babel-node server/index.js"
```
Comment (KvNGCzA) [inline package.json]: Please make this `start:dev` instead of `startDev`

File: .babelrc (hunk 1 above)
Comment (JuwonAbiola) [reply .babelrc]: ok done

File: package.json (hunk 2 above)
Comment (JuwonAbiola) [reply package.json]: done

File: package.json [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
+    "repository": "git@github.com:KvNGCzA/Hello-Books-BE.git",
+    "author": "KvNGCzA <akanmuoluwadamilare@yahoo.com>",
+    "license": "MIT",
+    "dependencies": {
+        "cors": "^2.8.5",
+        "dotenv": "^8.0.0",
+        "express": "^4.17.1",
+        "express-validator": "^5.3.1",
+        "morgan": "^1.9.1",
+        "nodemon": "^1.19.1"
```
Comment (KvNGCzA) [inline package.json]: please move nodemon under devDependencies

Comment (JuwonAbiola) [reply package.json]: ok

**Review by KvNGCzA — APPROVED** [review]

**Pull request merged**